workers/unity/Packages/com.improbable.gdk.mobile/Android/Utility/AndroidDeviceInfo.cs

```diff
@@ -2,22 +2,22 @@
 
 namespace Improbable.Gdk.Mobile.Android
 {
+    public enum AndroidDeviceType
+    {
+        Unknown,
+        Physical,
+        Emulator
+    }
+
     public static class AndroidDeviceInfo
     {
         public const string AndroidStudioEmulatorDefaultCallbackIp = "10.0.2.2";
```

Comment:
Also somewhat unrelated but if we are not calling the emulator `AndroidStudioEmulator` nowhere else, we can probably drop AndroidStudio from here. Considering it is the default emulator of choice I think we can go with that change.

Reply:
👍